tsconfig.json

```diff
@@ -9,7 +9,8 @@
     "target": "es5",
     "typeRoots": ["./node_modules/@types"],
     "experimentalDecorators": true,
-    "emitDecoratorMetadata": true
+    "emitDecoratorMetadata": true,
+    "esModuleInterop": true
```

Comment:
Why is this added?

Reply:
Hi @vojtechjelinek, the flag allows default imports from modules with no default export.  Importing from `lodash/cloneDeep` using following syntax: `import * as cloneDeep from 'lodash/cloneDeep';` results in a compilation error as follows: `This module can only be referenced with ECMAScript imports/exports by turning on the 'esModuleInterop' flag and referencing its default export.` So, I added this flag and switched to default import instead of using `* _ as` syntax.

Further, I tried to include the files in tsconfig and then use import statements without `.ts` extension but that results in an error of module not found.